docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+f. Run `kubectl cluster-info` to check if you are connected to the correct cluster.
+
+## Main actors
+
+Let's introduce the main actors that will lead us through the guides. These are our in-house examples, mock applications, and experimental services:
+
+- [`orders-service`](https://github.com/kyma-project/examples/tree/master/orders-service) is a sample microservice written in Go. It can expose HTTP endpoints that can be used to create and read basic order JSON entities. The service can run with either an in-memory database that is enabled by default or an external Redis database. You will see on its example how you can use Kyma to deploy your own microservice, expose it on HTTP endpoints to make it available for external services, and then bind it to an external database service (Redis).
+
+- [`orders-function`](https://github.com/kyma-project/examples/tree/master/orders-service/deployment/function.yaml) that is an equivalent of `orders-service` with the ability to retrieve order records and save data in it. Like the microservice, the Function can run with either an in-memory database or a Redis instance.
+
+- [Redis addon](https://github.com/kyma-project/addons/tree/master/addons/redis-0.0.3) is basically a bundle of two Redis services available in two plans: `micro` and `enterprise`. You will connect it to Kyma through Helm Broker and expose it in the Kyma cluster as an addon under Service Catalog. The Redis service represents an open-source, in-memory data structure store, used as a database, cache and message broker. For the purpose of these guides, you will use the `micro` plan with in-memory storage to demonstrate how it can replace the default memory of the microservice.
+
+- [Commerce mock](https://github.com/SAP-samples/xf-addons/tree/master/addons/commerce-mock-0.1.0) is to act as a sample external and monolithic application which we want to extend with Kyma. It is based on the [Varkes](https://github.com/kyma-incubator/varkes) project and is also available in the form of an addon. It will simulate how you can pair an external application with Kyma and expose the application's APIs and events. In these guides, you will use its `order.deliverysent.v1` event type to trigger both the microservice and the Function.
+
+## Steps
+
+These guides cover the following steps:
+
+1. [Create a Namespace](#getting-started-create-a-namespace) (`orders-service`) in which you will deploy all resources.
+2. [Create a microservice](#getting-started-deploy-a-microservice) by deploying `orders-service` on the cluster in the `orders-service` Namespace.
+3. [Expose the microservice](#getting-started-expose-the-microservice) through the APIRule CR on HTTP endpoints. This way it will be available for other services outside the cluster so you could retrieve data from it and send sample orders to it. Since the microservice uses in-memory storage, when you send a sample order to its endpoint, the order details will be gone after you restart the microservice.
```

Comment:
```suggestion
3. [Expose the microservice](#getting-started-expose-the-microservice) through the APIRule custom resource (CR) on HTTP endpoints. This way it will be available for other services outside the cluster so you can retrieve data from it and send sample orders to it. Since the microservice uses in-memory storage, when you send a sample order to its endpoint, the order details will be gone after you restart the microservice.
```
If the user is a beginner (and we should assume so, given that they're Getting Started guides), they might not be familiar with the acronyms we use.

Reply:
yes, sure :) I normally explain that always at the beginning of the doc, I forgot here so that's for noticing that.

Reply:
No worries :)